documentation/biblatex-manuscripts-philology.tex

```diff
@@ -98,17 +119,26 @@ \subsection{Optional}
 \fielditem{shortlibrary}{literal} the abreviated form of the library. Not used by the default style.
 
 
-\fielditem{pages}{range} the pages which are studied in the manuscript. In this field, you can use the macros \cs{recto} and \cs{verso}.
+\fielditem{pages}{range} the pages which are studied in the manuscript. In this 
+field, you can use the macros \cs{recto} and \cs{verso}.  This value will be used 
+when formatting citations.
+
+Use this field also for manuscript folio numbers.  E.g., “3\cs{verso}--5\cs{recto}.”
```

Comment:
Use \verb or \csquotes but not directl ""

Reply:
Sorry!  I wasn't able to compile the document, so errors like this slipped through my fingers.

Reply:
not a problem, I will correct. This compilation problem is quite strange, anyway